src/Agent/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Deprecations/Removed Features
 * The scriptable installers have been removed. ([#1170](https://github.com/newrelic/newrelic-dotnet-agent/pull/1170))
+* Castle.Monorail v2 instrumentation has been reomoved. ([#1177])(https://github.com/newrelic/newrelic-dotnet-agent/pull/1177)
```

Comment:
```suggestion
* Castle.Monorail instrumentation has been removed. ([#1177])(https://github.com/newrelic/newrelic-dotnet-agent/pull/1177)
```

I think that by specifying "v2" specifically, somebody might think that we still have instrumentation for other versions of Castle.Monorail, when we don't.
```suggestion
* Castle.Monorail v2 instrumentation has been reomoved. ([#1177])(https://github.com/newrelic/newrelic-dotnet-agent/pull/1177)
```